components/update_user_group_modal/update_user_group_modal.tsx

```diff
@@ -115,11 +115,12 @@ const UpdateUserGroupModal = (props: Props) => {
         } else {
             goBack();
         }
-    };
-    const goBack = () => {
+    }, []);
```

Comment:
I miss here  `name`, `mention`, `goBack`, `props.groupId` and maybe `props.actions.patchGroup`